Scenarios/AKS-Secure-Baseline-PrivateCluster/Terraform/04-network-hub.md

```diff
@@ -48,14 +52,24 @@ terraform plan -out $layerNametfstate -input=false -var="subscription_id=$ARM_SU
 terraform apply -var="subscription_id=$ARM_SUBSCRIPTION_ID" -var="tenant_id=$tenantId" -var="client_id=$servicePrincipalId" -var="client_secret=$servicePrincipalKey" -var="resource_group_name=$backendResourceGroupName" -var="storage_account_name=$backendStorageAccountName" -var="container_name=$backendContainername" -var="access_key=$layerNametfstate"
 ```
 
-After the domain controller is up and running, you will need to update the DNS on the vnet to use the domain controller for routing. 
+### Configure DNS on the hub network
+
+After the domain controller is up and running, you will need to update the DNS on the virtual network to use the domain controller for resolution.
 
-1. Locate the private IP address of your domain controller. (It should be a 10.X IP address)
-2. In the portal, navigate to Settings-> DNS servers
-3. Choose Custom, input your domain controller's IP address and hit Save. 
-4. Restart your jumpbox VM and domain controller to propagate the new DNS settings. 
+1. To check your domain controller is up, verify that the VM is in a running state by going to the Portal, clicking on the domain controller VM and verifying that the "Status" property says "Running". You should also verify that the Networking property "Private IP address" has a value. 
+2. Locate the private IP address of your domain controller. (It should be a 10.X IP address)
+   ```PowerShell
+    $nicId = (az vm show -g <Your Hub RG Name> -n <name of Domain Controller VM> --query "networkProfile.networkInterfaces[0].id")
+    az network nic show --id $nicId --query "ipConfigurations[0].privateIpAddress"
+   ```
+3. In the portal, navigate to Settings-> DNS servers
+4. Choose Custom, input your domain controller's IP address and hit Save. 
+5. Restart your domain controller first and then the jumpbox VM to propagate the new DNS settings. Default values for the network resource group and VM names used in example below. 
+   ```PowerShell
+   az vm restart -g "aks-HUB" -n "svr-dev-dc"
+   az vm restart -g "aks-HUB" -n "server-dev-linux"
```

Comment:
Any reason you were able to hardcode `aks-HUB` here, but in the command above you needed to use `<Your Hub RG Name>`?  If we can accurately hardcode in the command above, please do.  It'll minimize human error.